examples/partially_blindrsa.ts

```diff
@@ -9,7 +10,7 @@ function hexNumToB64URL(x: string): string {
     if (x.startsWith('0x')) {
         x = x.slice(2);
     }
-    return sjcl.codec.base64url.fromBits(sjcl.codec.hex.toBits(x));
+    return Buffer.from(x, 'hex').toString('base64url');
```

Comment:
consider https://gist.github.com/thibmeu/d49746a77b09a65807801e92a50cbba4 for hex to base64url conversion

Reply:
done